src/ltv/graph.py

```diff
@@ -0,0 +1,104 @@
+import logging
+
+from eth_typing import ChecksumAddress
+from gql import gql
+from hexbytes import HexBytes
+from web3 import Web3
+from web3.types import Wei
+
+from .clients import graph_client
+from .typings import HarvestParams
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+DEFAULT_NETWORK_ID = '0'
+
+
+def graph_get_ostoken_vaults() -> list[ChecksumAddress]:
+    query = gql(
+        """
+        query OsTokenVaultsIds($network: String) {
+          networks(
+            where: { id: $network }
```

Comment:
Don't think you need "where" here, just take the first element from array. There  will always be only one network object.